app/models/validations/sales/soft_validations.rb

```diff
@@ -60,7 +60,7 @@ def savings_over_soft_max?
   end
 
   def deposit_over_soft_max?
-    return unless savings && deposit
+    return unless savings && deposit && mortgage_used?
```

Comment:
do we want to apply this change to the soft min, too? (an/or the hard max/min?)

Reply:
I don't think there is a soft min or hard validations and the ticket specifically points of this calculation `deposit > savings * 4 / 3` so I think it's just this one